signer/metering_auth.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+from web3.auto import w3
+from common.repository import Repository
+from signer.constant import StatusCode, StatusMessage
+
+required_data = ['demon_id', 'message_data_', 'signature']
+
+
+def extract_public_key(message_data, signature):
+    public_key = w3.eth.account.recoverHash(message_data, signature=signature)
+    return public_key
+
+
+def make_response(status_code, body, header=None):
+    return {
+        "statusCode": status_code,
+        "headers": header,
```

Comment:
put this function in common folder , as these can be used by other services  as well.